component-definition.json

```diff
@@ -150,6 +150,37 @@
               }
             }
           }
+        },
+        {
+          "title": "Curated Cards",
+          "id": "curated-cards",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
+                "template": {
+                  "name": "Curated Cards",
+                  "model": "curated-cards",
```

Comment:
Just for my understanding. Is the `model` name required here?

Reply:
@thomsebastin , It is required as its map the authoring to block name in blocks folder where JS and CSS code resides